wordpress-mobile/WordPress-Android
# Experimental page template selector

See https://github.com/wordpress-mobile/gutenberg-mobile/pull/1568

To test:

Launch the app and ensure the template picker is shown only on empty pages and when metro is running (__DEV__ == true)

PR submission checklist:
